store/extensions/StickiesArt/extension.json

```
@@ -28,7 +28,7 @@
         "clients": ["Flash"]
     },
 
-    "submissionDate": "30-06-2024 19:50:10",
-    "updateDate": "30-06-2024 19:50:10",
+    "submissionDate": "04-07-2024 17:17:10",
+    "updateDate": "04-07-2024 17:17:10",
```

Only update the update date, not the submission date
```suggestion
    "submissionDate": "30-06-2024 19:50:10",
    "updateDate": "04-07-2024 17:17:10",
```